openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -23,6 +23,10 @@
     EMAIL_MIN_LENGTH, EMAIL_MAX_LENGTH, PASSWORD_MIN_LENGTH, PASSWORD_MAX_LENGTH,
     USERNAME_MIN_LENGTH, USERNAME_MAX_LENGTH
 )
+from .helpers import (
+    get_profile_image_names as _get_profile_image_names,
```

Comment:
Can we just move these methods out of "helpers"? I don't see the point in the current structure, since this file has proxies to these methods.